### PR TITLE
Fixes Issue 269 - Numbering to use CIS Versions

### PR DESCRIPTION
**Pull request by robertojrojas**
Fixes Issue #269 - Numbering to use CIS Versions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,7 +202,40 @@ go build -o kube-bench .
 
 ## Running on OpenShift 
 
-kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10` when you run the `kube-bench` command (either directly or through YAML). This config version is valid for OCP 3.10 and 3.11. 
+| OpenShift CIS Benchmark | kube-bench config |
+|---|---|---|
+| ocp-3.10| rh-0.7 |
+| ocp-3.11| rh-0.7 |
+
+kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10`.
```
Comment (lizrice) [inline README.md]:
```suggestion
kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--benchmark rh-07`, or `--version ocp-3.10` or `--version ocp-3.11`
```

File: README.md [hunk 2]
```diff
@@ -183,7 +202,40 @@ go build -o kube-bench .
...
 
-kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10` when you run the `kube-bench` command (either directly or through YAML). This config version is valid for OCP 3.10 and 3.11. 
+| OpenShift CIS Benchmark | kube-bench config |
+|---|---|---|
+| ocp-3.10| rh-0.7 |
+| ocp-3.11| rh-0.7 |
+
+kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10`. 
+
+When you run the `kube-bench` command (either directly or through YAML). This config version is valid for OCP 3.10 and 3.11.
```
Comment (lizrice) [inline README.md]:
```suggestion
when you run the `kube-bench` command (either directly or through YAML). 
```

The table already makes the point about this config being valid for 3.10 / 3.11

File: README.md [hunk 3]
```diff
@@ -183,7 +202,40 @@ go build -o kube-bench .
...
+```
+
+
+`controls` are also organized by Red Hat's OpenShift hardening guide version under the `cfg` directory for
+example `cfg/rh-0.7`.
+
+
+
+**Note:**  **`It is an error to specify both --version and --benchmark flags together`**
+
```
Comment (lizrice) [inline README.md]: I don't really think we need to repeat this again, it is covered enough above, so i think we could just delete it. But also note that this is talking about Red Hat's OpenShift hardening guide which is not a CIS Benchmark, so when we're talking about OpenShift it's wrong to say CIS Benchmark.

**Comment by robertojrojas** [comment]
Hopefully merges with old **cfg** files is not too crazy.

**Pull request merged**